src/Header.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+export default function Header() {
+    return (
+        <>
```

Comment:
I don't think we'll need those fragments here as the elements are within the `Container`